docs/user-guide/config-examples.md

```diff
@@ -0,0 +1,43 @@
+# User configurations
+
+Want some help with starting your configuration?
+Or maybe you're curious how others have done things?
+Take a look at these configuration examples below.
+
+<!-- TODO: add info about `kickstart.nixvim` -->
+
+**Note:**\
```

Comment:
Indeed, But I was finding it clearer to have this header against the text block than breaking a line.
But I can add a blankine if you prefer.

Reply:
> But I was finding it clearer to have this header against the text block than breaking a line.

No, I was trying to say that most markdown flavors would render this the same with or without escaping that LF.

Is that not the case with mdbook?

Reply:
Actually, reading the spec having the `\` does the opposite from what I thought it did (I thought it was a redundant escape sequence):

> For a more visible alternative, a backslash before the [line ending](https://github.github.com/gfm/#line-ending) may be used instead of two spaces:\
> [Example 659](https://github.github.com/gfm/#example-659)
>
> ```markdown
> foo\
> baz
> ```
>
> ```html
> <p>foo<br />
> baz</p>
> ```



> But I was finding it clearer to have this header against the text block than breaking a line.

I think I misinterpreted this. Were you saying you preferred rendering a linebreak after the **Note:** header, but wanted to avoid a full _paragraph_ break?

Your current code should render something like this:

```html
<p>
<b>Note:<b>
<br>
Most of those configurations are using a <a href="../modules/standalone.html">standalone build</a>, however,
all of the nixvim options are accessible no matter how you are using it (flake, NixOS/HM module, nix-darwin...).
</p>
```

or

**Note:**\
Most of those configurations are using a [standalone build](../modules/standalone.html), however, all of the nixvim options are accessible no matter how you are using it (flake, NixOS/HM module, nix-darwin...).

Reply:
(This is resolved, assuming this is rendering as you intended)